### PR TITLE
#304 groundwork for legacy content css styling

### PR DESCRIPTION
**Pull request by mrblippy**

### CODE REVIEW

**Comment by PenghaiZhang** [comment]
SearchQueryProperties test is broken. This one is not a flakey test so need to have a look what breaks it.

**Comment by mrblippy** [comment]
@doolse Yeah, that's a good idea

File: Source/Plugins/Core/com.equella.core/js/package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "bundleps:main": "cross-env-shell \"purs bundle \"${npm_package_config_psglob}\" -m Bridge --main Bridge -o entrybuild/target/main.js\"",
     "bundleps:selection": "cross-env-shell \"purs bundle \"${npm_package_config_psglob}\" -m OEQ.SelectionUI.Main --main OEQ.SelectionUI.Main -o entrybuild/target/selection.js\"",
     "bundleps:uploadlist": "cross-env-shell \"purs bundle \"${npm_package_config_psglob}\" -m Uploads.UploadList --main Uploads.UploadList -o entrybuild/target/uploadlist.js\"",
-    "bundleps:langbundle": "cross-env-shell \"purs bundle \"${npm_package_config_psglob}\" -m Tools.GenLangStrings --main Tools.GenLangStrings -o target/genlang.js\""
+    "bundleps:langbundle": "cross-env-shell \"purs bundle \"${npm_package_config_psglob}\" -m Tools.GenLangStrings --main Tools.GenLangStrings -o target/genlang.js\"",
+    "scss": "sass ../resources/web/css/saas/legacy.scss ../resources/web/css/saas/output/legacy.css"
```
Comment (doolse) [inline Source/Plugins/Core/com.equella.core/js/package.json]: You don't need this and resources/web/css/saas/legacy.scss file now that your using parcel anymore i believe?

**Comment by mrblippy** [comment]
Closing this one, due to changing approach of performing sass compilation on the server side to integrate with new theme selector. New pull request to come..

**Pull request closed without merging**